susemanager/src/mgr_bootstrap_data.py

```diff
@@ -571,6 +571,10 @@
     "venv-salt-minion",
 ]
 
+PKGLISTFUSS12 = [
```

Comment:
Shouldn't you reuse `PKGLISTDEBIAN12`?

Reply:
We did not want to add anything other than the Salt Bundle to new OS's, right?

Reply:
That's right, but the other things maybe needed in very basic debian installations as maybe they are not installed by default in such cases, and that could be the reason they're there... and could be needed for FUSS12 as well.

Reply:
Suggestion accepted, changed adding all the missing packages, now it's the same as Debian 12 (or Raspberry Pi OS 12).